app/models/host_content_item.rb

```diff
@@ -36,6 +36,18 @@ def for_document(document, page: nil, order: nil)
         total_pages: api_response["total_pages"],
         rollup: rollup(api_response),
       )
+    rescue GdsApi::HTTPNotFound
+      HostContentItem::Items.new(
```

Comment:
I think it might be cleaner to return `nil`, then add a guard in the view so we don't display the host content detail if `nil` is returned

Reply:
Yeah, fair enough. We can iterate as we go - I agree, pragmatism is the way forward!